lib/language-dataset.js

```diff
@@ -156,10 +156,10 @@ export default class LanguageDataset {
    */
   static checkMatches (featureList, inflection, item, comparisonType = Morpheme.comparisonTypes.EXACT) {
     let fullMatchQty = featureList.length
-    let matches = featureList.reduce((acc, f) => {
-      if (inflection.hasOwnProperty(f) && item.features.hasOwnProperty(f) && item.featureMatch(inflection[f], comparisonType)) {
+    const matches = featureList.reduce((acc, f) => {
+      if (inflection.hasOwnProperty(f) && item.features.hasOwnProperty(f) && item.featureMatch(inflection[f], comparisonType)) { // eslint-disable-line no-prototype-builtins
```

Comment:
`// eslint-disable-line no-prototype-builtins`

Am I right - it is not a comment - it is a eslint exception?
may be it is worth to add it to the whole code once, wht do you think?

Reply:
Yes, that's an exception for [the following ESLint rule](https://eslint.org/docs/rules/no-prototype-builtins). This rule makes sense to me although I do not see it applied often, even MDN does use `hasOwnPropety` directly in its examples https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty (i.e. `someObject.hasOwnProperty()` instead of `someObject.prototype.hasOwnProperty.call(foo, "bar")`), but I was not mentally prepared to apply it everywhere 🙂. On my opinion, even with all advantages, it makes code harder to read. It might also be leading to more trickier and subtle side effects than just the replacement of `lets` into `consts`. So I thought it's safer to leave such changes for later. I'm just not ready to jump into this yet 🙂. What is your opinion on the issue?

Do you suggest we add an exception for this rule into a `package.json` ESLint config instead of specifying it on a line by line bases? @balmas, what do you think?

Reply:
I don't feel strongly about it. I can see an argument either way. I'm fine with it line-by-line for now.